inst/htmljs/animint.js

```diff
@@ -2053,6 +2071,28 @@ var animint = function (to_select, json_file) {
     ////////////////////////////////////////////
     // Widgets at bottom of page
     ////////////////////////////////////////////
+
+    // Function to start the tour
+  var element = d3.select("body");
+
+  function startTour() {
+    const driver = window.driver.js.driver;
+
+    const driverObj = driver({
+      showProgress: true,
+      steps: steps
+    });
+
+    driverObj.drive();
+  }
+ 
+  element.append("button")
+  .text("Start Tour")
+   
+  .on("click", function() {
+    startTour();
```

Comment:
if this is the only place that startTour is called, I think it not worth adding the startTour function. can you please move the contents of startTour inside this anonymous function?